src/databricks/labs/ucx/install.py

```diff
@@ -127,6 +127,7 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
             functools.partial(table, "directfs_in_queries", DirectFsAccess),
             functools.partial(table, "used_tables_in_paths", UsedTable),
             functools.partial(table, "used_tables_in_queries", UsedTable),
+            functools.partial(table, "source_table_grants", Grant),
```

Comment:
this name is suboptimal. how can we name it better?

Reply:
OK